packages/react-components/src/components/SendBox.tsx

```diff
@@ -216,52 +259,57 @@ export const SendBox = (props: SendBoxProps): JSX.Element => {
-        inlineChildren={true}
-        disabled={disabled}
-        errorMessage={onRenderSystemMessage ? onRenderSystemMessage(errorMessage) : errorMessage}
-        textFieldRef={sendTextFieldRef}
-        id="sendbox"
-        inputClassName={sendBoxStyle}
-        placeholderText={strings.placeholderText}
-        textValue={textValue}
-        onChange={setText}
-        onKeyDown={() => {
-          onTyping && onTyping();
-        }}
-        onEnterKeyDown={() => {
-          sendMessageOnClick();
-        }}
-        styles={mergedStyles}
-        supportNewline={supportNewline}
-        maxLength={MAXIMUM_LENGTH_OF_MESSAGE}
-      >
-        <InputBoxButton
-          onRenderIcon={onRenderSendIcon}
-          onClick={(e) => {
-            if (!textValueOverflow) {
-              sendMessageOnClick();
-            }
-            e.stopPropagation();
-          }}
-          id={'sendIconWrapper'}
-          className={mergedSendButtonStyle}
-          ariaLabel={localeStrings.sendButtonAriaLabel}
-          tooltipContent={localeStrings.sendButtonAriaLabel}
-        />
-      </InputBoxComponent>
+    <Stack className={mergeStyles(sendBoxWrapperStyles)}>
       {
-        /* @conditional-compile-remove(file-sharing) */
-        props.onRenderFileUploads && props.onRenderFileUploads()
+        /* @conditional-compile-remove-from(stable): FILE_SHARING */
+        renderFileUploadErrorMessage
       }
+      <Stack
+        className={mergeStyles(
+          borderAndBoxShadowStyle(theme, isDarkThemed(theme) ? '#f1707b' : '#a80000', !!errorMessage, !!disabled)
+        )}
+      >
+        <InputBoxComponent
+          autoFocus={autoFocus}
+          data-ui-id={ids.sendboxTextField}
+          inlineChildren={true}
+          disabled={disabled}
+          errorMessage={onRenderSystemMessage ? onRenderSystemMessage(errorMessage) : errorMessage}
+          textFieldRef={sendTextFieldRef}
+          id="sendbox"
+          inputClassName={sendBoxStyle}
+          placeholderText={strings.placeholderText}
+          textValue={textValue}
+          onChange={setText}
+          onKeyDown={() => {
+            onTyping && onTyping();
+          }}
+          onEnterKeyDown={() => {
+            sendMessageOnClick();
+          }}
+          styles={mergedStyles}
+          supportNewline={supportNewline}
+          maxLength={MAXIMUM_LENGTH_OF_MESSAGE}
+        >
+          <InputBoxButton
+            onRenderIcon={onRenderSendIcon}
+            onClick={(e) => {
+              if (!textValueOverflow) {
+                sendMessageOnClick();
+              }
+              e.stopPropagation();
+            }}
+            id={'sendIconWrapper'}
+            className={mergedSendButtonStyle}
+            ariaLabel={localeStrings.sendButtonAriaLabel}
+            tooltipContent={localeStrings.sendButtonAriaLabel}
+          />
+        </InputBoxComponent>
+        {
+          /* @conditional-compile-remove-from(stable): FILE_SHARING */
```

Comment:
this has changed to `@conditional-compile-remove(file-sharing)` more info: https://teams.microsoft.com/l/message/19:5e150d250f724d1089c949305efdc8cc@thread.skype/1645837170611?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=3e9c1fc3-39df-4486-a26a-456d80e80f82&parentMessageId=1645812118780&teamName=Azure%20Communication%20Services&channelName=Web%20UI%20-%20Team&createdTime=1645837170611

Reply:
Done